tests/scripts/all.sh

```diff
@@ -2114,6 +2115,7 @@ component_test_valgrind_constant_flow () {
     scripts/config.py full
     scripts/config.py set MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND
     scripts/config.py unset MBEDTLS_USE_PSA_CRYPTO
+    scripts/config.py --force set MBEDTLS_VALGRIND
```

Comment:
Since this is solely about testing, the option should be called `MBEDTLS_TEST_xxx`.